### ISSUE TITLE
Add Kwok in Package Manger

### ISSUE THREAD

**Issue by wzshiming** [kubernetes-sigs/kwok#248]
### What would you like to be added?

- [x] brew Homebrew/homebrew-core#120889
- [ ] choco
- [ ] winget
- ...

### Why is this needed?

easy to use

**Comment by utkarsh-singh1** [comment]
Hi @wzshiming , Should we add KWOK to Choco as we have already added it to winget ( Also it is officialy supported plateform)?

**Comment by utkarsh-singh1** [comment]
> Yes, it's on the list.

Sorry for the confusing question , was actually wondering why should we have choco if we are already have KWOK on winget , also can we add an package monitoring tool like https://repology.org/?

**Comment by utkarsh-singh1** [comment]
Also, should we add more linux-distro based package managers , where KWOK can be added?

**Comment by wzshiming** [comment]
Yep, this has been completed. [see more](https://kwok.sigs.k8s.io/docs/user/github-action/)

For linux-distro, it seems that KInD also does not provide for it [KInD installation](https://kind.sigs.k8s.io/docs/user/quick-start/#installation), if there is an easy way to add it I personally also like it.

**Comment by utkarsh-singh1** [comment]
Maybe kwok can have kubectl like installation.

**Comment by utkarsh-singh1** [comment]
Or there can be a appimage for the kwok and kwokctl executable included in one image.

**Comment by wzshiming** [comment]
> Maybe kwok can have kubectl like installation, will try to raise a package resource for fedora.

Sounds good, how do I do it?

> Or there can be a appimage for the kwok and kwokctl executable included in one image.

Do you mean this one? https://kwok.sigs.k8s.io/docs/user/all-in-one-image/
